### DISCUSSION TITLE
Column facet: how to align header title to the bottom?

### DISCUSSION THREAD

**Discussion by aborruso** [vega/vega-lite#8722] [Q&A]
Hi,
I would like to align to the bottom all the header titles of my facet columns.

[This is my code](https://vega.github.io/editor/#/url/vega-lite/N4Igxg9gdgZglgcxALlANzgUwO4tJKAFzigFcJSBnAdTgBNCALFAZgAY2AacaYsiygAlMiRoVYcAvtxgBDMJnGoQlAA7ySSZAEY2k6SDqzCsvCCiyAtphSHjsgLSYARgCYXAFgCc2gBwBWVw8wLzZfL39-GAB2aK9oumig-1kOEAM5BSVQeEwAGzpbGBBuRkxZOkwAJzM82Wd8gDFeAGU4AC8bZFcuEDqGvIB5KqwiW2cIQkIIS3TuYkI8rpASkEIAT1VlqBmSWTy5lS2wWtl16pQAbVBLWSqAazMNrds7k+4KQmqAJQq4KhQuj03EwUEgdE0ZkgeQgNWUuQKtke3CWCFBhWUCyWtkOlFh2RACIx5kOz22uwsBwMTEUpnhWERyBAaFWlBMYEeyEIVVImHmm2WAEdSLIiHATMQ0DZ9NIbndOaAybYvgAPcQyOB5A5M5x1DmrKp-AE6bT+Ayg8GQ5TQ2FmIlI1ao9FPcXYpm4-F2hnEqCkgW2HaWPZU+aYNVe-LElkyWG3JQlAB02mK-JeTOFooWxjgUtJZRMEcZzNZ7M53N5qaFIrFEpz0v0AF1SiIEGJAb5eth6Ex28CQFVMHi8rnlJQwPsuor83SQCRKlsoJUxvpuAASMdlW62MSEVSUZAAegPUoQsgTCHFjFIzgTcAgB43mFux8wp4ceXFmGP-gTrgTbATAArPFfW4IwTEoRRKDMcDHBcdxnG8PxAmCUJwkiGI4gSJIPBSNJkGuEkdBkWwADloAAAmaAc2VWTkQFI1YWWQFg-x6bRtA8AA2cI2G0LjWO0WUiNcEimXIqAqNhQd1RAeiAFEmJQVw-3YSIPBYdhkkiVxhN9FixIYyjqJkujbBaJSdA8BMWH8MJ2GiLi+N8VxWK4vSUA8QyJKkmjZPo6hLJYbQkw8II4jYDxom0Lxwg8DzkH8QyTNo7h6MY7hmKCBNAn8aJcNcPKCqKhKuOS6TUrk2xFMy5SAO4thokipymoapIEuicq-LMpkLNq7oQo4VxoqGka2GGhLfC60y0tsQL+pYay+LCDxlt8VbdHWhKvEMgBVME6koSg4FyQpZvEyzXC8BMNNiNguKSfxuICRyEt0PaDtkI6TqwM6qqZGrixYv97LiYbXNcAT-CEzhQH0jiPrAQ7jtOnqQD6oH+Ju4bol8FhQiuxz1rNWGiO0UTCVsfaka+lHfrR+agZU-87OiriuI8FyfEayQG30IA)

And I get the below result, the titles are not aligned downwards, each one has a different distance from the graphic bottom.

![image](https://user-images.githubusercontent.com/30607/219201079-55747584-183b-4487-b006-c3a79f15a43a.png)

How to align these titles to the bottom?

Thank you

**Comment by PBI-David** [comment]
As discussed on SO, this looks like a bug. Work around  has been provided here: https://stackoverflow.com/questions/75488910/column-facet-how-to-align-header-title-to-the-bottom

**Reply by aborruso** [reply]
If I remove the height, it works
